lib/components/narrative/default/format-duration.js

```diff
@@ -0,0 +1,17 @@
+import moment from 'moment-timezone'
```

Comment:
Since this component is also used in other kinds of itineraries that aren't the default, this should live somewhere else, perhaps even in some common components folder.

Reply:
I moved this and format-time to a components/util/ folder in later [i18n-ramp-up](https://github.com/opentripplanner/otp-react-redux/pull/408) commits (https://github.com/opentripplanner/otp-react-redux/pull/408/commits/7600b0cd5e97242863603b0de3ac223e60318eb)

Reply:
moved in https://github.com/opentripplanner/otp-react-redux/pull/430/commits/ad8ea3326848a5ac077e01898d7f7657412fda84